receiver/datadogreceiver/receiver.go

```diff
@@ -61,30 +56,21 @@ func newDataDogReceiver(config *Config, nextConsumer consumer.Traces, params rec
 }
 
 func (ddr *datadogReceiver) Start(_ context.Context, host component.Host) error {
-	ddr.startOnce.Do(func() {
-		ddr.shutdownWG.Add(1)
-		go func() {
-			defer ddr.shutdownWG.Done()
-
-			ddmux := http.NewServeMux()
-			ddmux.HandleFunc("/v0.3/traces", ddr.handleTraces)
-			ddmux.HandleFunc("/v0.4/traces", ddr.handleTraces)
-			ddmux.HandleFunc("/v0.5/traces", ddr.handleTraces)
-			ddr.server.Handler = ddmux
-			if err := ddr.server.ListenAndServe(); err != http.ErrServerClosed {
-				host.ReportFatalError(fmt.Errorf("error starting datadog receiver: %w", err))
-			}
-		}()
-	})
+	go func() {
+		ddmux := http.NewServeMux()
+		ddmux.HandleFunc("/v0.3/traces", ddr.handleTraces)
+		ddmux.HandleFunc("/v0.4/traces", ddr.handleTraces)
+		ddmux.HandleFunc("/v0.5/traces", ddr.handleTraces)
```

Comment:
Is this change here competing with the one adding support for 0.7?

Reply:
A conflict is inevitable